tests/functional/features/application_restricted.feature

```diff
@@ -124,7 +124,7 @@ Feature: Unattended Access
     And I create a new app
     And I add the attribute with key of apim-app-flow-vars and a value of { "pds" : { "app-restricted": { "update": true } }}
     And I add the scope urn:nhsd:apim:app:level3:personal-demographics-service
-    And I wait for 500 milliseconds
+    And I wait for 200 milliseconds
```

Comment:
I previously added the extra time to reduce intermittent test failures. If you've had smooth runs through the pipelines with these changes, then feel free to keep them as a lower value (especially if we are rerunning the tests an increased number of times).

Reply:
The application restricted tests seem to work fine (at the moment) with the shorter wait time, but I'll keep this in mind if they start failing in the future.